examples/per_region_images.yaml

```diff
@@ -2,8 +2,8 @@ resources:
   cloud: aws
   instance_type: g4dn.xlarge
   image_id: 
-    us-west-2: ami-0fe5af21074ad2a10  # Deep learning AMI with CUDA 11.6 without conda installed
-    us-west-1: skypilot:gpu-ubuntu-1804
+    us-west-2: skypilot:gpu-ubuntu-1804
+    us-east-2: ami-0406ab83559331633 # AWS Deep Learning AMI GPU CUDA 11.4.3 (Ubuntu 20.04) 20220210 without conda installed
```

Comment:
Why are we changing the regions here?

Reply:
The previous setting does not work with the AWS tests. These are patches for #1587. The main reason is that my account (I believe many AWS account) does not have GPU quota on us-west-1.

Reply:
Got it. BTW, I have some T4 quotas in the region. Apply for G4 instances! 😃